### ISSUE TITLE
Contrast on hover

### ISSUE THREAD

**Issue by lramarojaona** [openfed/AccessibilityCheck#8]
Hello @bdeclerc ,

The WCAG 2.0 AA requires a sufficient contrast when hovering links with the mouse pointer.
Currently the tool doesn't provide such reports.

Can we have your feedback on this?

Thanks!

**Comment by bdeclerc** [comment]
WCAG2.0 AA requires a lot of things the tool doesn't provide a report for (many because they can't be tested by a script but need to be testen by a human)

This one looks like it could be a good addition to the scripted checks, I'll label it "enhancement" and assign to me for now.

**Comment by stefanruijsenaars** [comment]
Implemented in https://github.com/openfed/AccessibilityCheck/pull/14

This applies the same check as it does for regular text - but also checking the :hover state (it won't be 100% accurate as there's no clean way of accessing the :hover state in javascript https://stackoverflow.com/questions/11495535/why-doesnt-getcomputedstyle-work-with-pseudo-classes-like-hover - instead we use the magic in https://stackoverflow.com/a/22638396)

If it detects insufficient contrast (and the element is not underlined - i.e. we rely on colour alone to show that this is a link), it will show a warning.

See also:
- https://webaim.org/blog/wcag-2-0-and-link-colors/
- https://github.com/w3c/wcag/issues/400
